flathub.json

```diff
@@ -0,0 +1,3 @@
+{    
+    "only-arches": ["x86_64"]
```

Comment:
I had comments: why is this x86_64 only?

Reply:
This constraint might indeed not be needed.